suite-native/module-home/src/components/GraphBalance.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
Please no :D This was on purpose in one file, these components are meant to be used only in that header so they are not exported, so no one will be able to use then anywhere else and they are super small so they definitely belongs there. This is only adding noise to file system and when I need to tweak this header I have to switch between 4 four files like crazy.

Reply:
Maybe just place atoms to components files and that will make them worthy enough be standalone. Let's see how it will looks.